src/Hashing.purs

```diff
@@ -54,3 +58,9 @@ datumHash =
 plutusScriptHash :: PlutusScript -> Aff (Maybe ScriptHash)
 plutusScriptHash =
   map (scriptHashFromBytes <<< wrap) <<< Promise.toAffE <<< hashPlutusScript
+
+-- | Calculates the hash of the transaction by applying `blake2b256Hash` to
+-- | the cbor-encoded transaction body.
+transactionHash :: Serialization.Transaction -> Aff TransactionHash
```

Comment:
I think there are a few other places where we've used `toBytes <<< asOneOf <<< ...` for `Transaction`s before. Maybe we can replace those now?

Reply:
There doesn't seem to be any other places in the codebase where we need transaction hash calculation